package.json

```diff
@@ -3,7 +3,7 @@
   "productName": "Beet Blockchain Companion",
   "version": "0.2.0-rc1",
   "apiversion": 3,
-  "description": "Beet is a stand-alone key-manager and signing app for blockchains like BitShares, Steem and EOS, as well as BTC, BinanceChain and others.",
+  "description": "Beet is a stand-alone key-manager and signing app for blockchains like BitShares, Steem and others.",
```

Comment:
Maybe only mention BitShares? I'm not sure if other chains are supported or if we have resources to work on supporting them.

Reply:
They're supported already, just less maintained than bitshares.

Reply:
RE multiple chains: https://github.com/bitshares/beet/issues/196

You're right though, whilst they can be or have been supported in the past, their inclusion may only bring passive benefits associating with larger cap cryptos, we don't yet see any use of BEET in EOS/BNB/WLS despite being supported for 2+ years

Reply:
And Steem? Note we posted progresses on Hive only.

Reply:
Yeah, the blockchain config is located: https://github.com/bitshares/beet/blob/develop/src/config/config.js

Supports BTS, BTS_TEST, TUSC, BTC, BTC_TEST in current dev branch.

Currently disabled: Steem, Whaleshare, EOS, TLOS, BNB, BNB_TEST

We do indeed post primarily on hive, I feel there's greater association given blocktrades hive dev involvement. Supporting Hive would likely be as simple as adding TUSC was.

Reply:
Thanks.

To be clear, I'm not a big fan of supporting other chains in Beet at the moment, even though I'm personally somewhat involved in Hive.

Since we have disabled Steem in the develop branch already, but kept Bitcoin, I suggest we update the sentence from
`... for blockchains like BitShares, Steem and others.`
to
`... for blockchains like BitShares, Bitcoin and others.`

Reply:
Done